cppcache/integration/framework/Cluster.cpp

```diff
@@ -89,7 +90,7 @@ void Server::stop() {
   started_ = false;
 }
 
-void Cluster::start() {
+void Cluster::start(std::function<void()> fn) {
```

Comment:
I would feel a little bit better if we could name `fn` to represent what kind of function we intend to pass in here (but not feeling too strongly about).

Reply:
I agree. fn is never a good name for a function.  Renaming to extraGfshCommands().